ansible_collections/arista/cvp/plugins/modules/cv_validate_v3.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+#
+# Copyright 2023 Arista Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Required by Ansible and CVP
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: cv_validate_v3
+version_added: "3.7.0"
+author: Ansible Arista Team (@aristanetworks)
+short_description: Online/Offline configlet validation
+description:
+  - CloudVision Portal Validate module to Validate configlets against a device on CVP
+options:
+  device:
+    description: CVP device information
+    required: true
+    type: list
+    elements: dict
```

Comment:
Since this is actually a dictionary that includes device reference (hostname, serial etc) and the configlets, I propose we name this variable. something other than `device`

Reply:
I named this `device` because it contains information about the device and the configlet that needs to be validated against it.
Do you have a better name in mind?
@chetryan @noredistribution

Reply:
`device` or `device_id` is fine, we should also update the schema as well then no?

Reply:
Discussed during the call to rename to `devices` since it's a list of dictionaries.